src/ui/components/services/storage.ts

```diff
@@ -0,0 +1,21 @@
+import Todo from '../models/todo';
```

Comment:
I believe this should be in either `src/ui/components/todo-item/-services/storage.ts` (if it is "private" to this component).